modules/site-navigation/module.php

```diff
@@ -30,4 +30,17 @@ public function __construct() {
 	public function get_name() {
 		return 'site-navigation';
 	}
+
+	/**
+	 * Determine whether the module is active.
+	 *
+	 * @return bool
+	 */
+	public static function is_active() {
+		if ( defined( 'ELEMENTOR_TESTS' ) && ELEMENTOR_TESTS ) {
```

Comment:
```suggestion
		if ( Utils::is_elementor_tests() ) {
```
And add Utils to the use blocks at the top.